x-pack/plugins/security_solution/public/detection_engine/rule_management/api/hooks/use_bulk_schedule_backfill_mutation.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { IHttpFetchError } from '@kbn/core-http-browser';
+import type { UseMutationOptions } from '@tanstack/react-query';
+import { useMutation } from '@tanstack/react-query';
+import type { BulkScheduleBackfillActionResponse } from '../../../../../common/api/detection_engine';
+import { DETECTION_ENGINE_RULES_BULK_ACTION } from '../../../../../common/constants';
+import type {
+  BulkScheduleBackfillActionErrorResponse,
+  PerformBulkScheduleBackfillActionProps,
+} from '../api';
+import { performBulkScheduleBackfillAction } from '../api';
+
+export const BULK_ACTION_MUTATION_KEY = ['POST', DETECTION_ENGINE_RULES_BULK_ACTION];
```

Comment:
If we are keeping this mutation hook separate from `useBulkActionMutation`, we should modify the `BULK_ACTION_MUTATION_KEY` value that we use for the mutation key to avoid unwanted caching invalidations with the other hook, which uses exactly the same key.

I think something like:
```ts
export const BULK_SCHEDULE_BACKFILL_MUTATION_KEY = ['POST', DETECTION_ENGINE_RULES_BULK_ACTION, 'SCHEDULE_BACKFILL'];
```
should suffice.

Reply:
removed this mutation hook